.github/release-drafter.yml

```diff
@@ -0,0 +1,37 @@
+# https://github.com/release-drafter/release-drafter?tab=readme-ov-file#configuration-options
+template: |
+  $CHANGES
+
+  **Full Changelog**: https://github.com/$OWNER/$REPOSITORY/compare/$PREVIOUS_TAG...v$RESOLVED_VERSION
+name-template: 'docs-builder v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+change-template: '- $TITLE by @$AUTHOR in #$NUMBER'
+categories:
+  - title: '💥 Breaking Changes'
+    labels:
+      - 'changelog:breaking' # When a breaking change is introduced
+  - title: '✨ Features'
+    labels:
+      - 'changelog:feature' # When a new feature is introduced
+      - 'changelog:enhancement' # When an existing feature is improved
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'changelog:fix' # When a bug is fixed
+  - title: '📝 Documentation'
+    labels:
+      - 'changelog:docs' # When documentation is updated
+  - title: '🧰 Maintenance'
+    labels:
+      - 'changelog:chore' # When a chore is done
+      - 'changelog:ci' # When CI is updated
+      - 'changelog:dependencies' # When dependencies are updated
+exclude-labels:
+  - 'changelog:skip' # When a PR should be excluded from the changelog
```

Comment:
We can still discuss the appropriate labels.

Reply:
ahh ignore previous comments, can we remove the `changelog:` prefix from these?

Reply:
Updated labels: https://github.com/elastic/docs-builder/pull/202/commits/e49c1c73089d99ae8ef78bd74068b44a158f55a0